### PR TITLE
Add an option to link to Couchers.org on profile

### PR DESCRIPTION
**Pull request by aapeliv**
#### Proposed Changes

* As discussed on Slack, adds an option to link to Couchers.org on profile

Unfortunately I tried a few different ways of running the app locally, but I failed to get it running... So this is a bit of an "armchair" contribution... I'd appreciate someone checking over it or testing it locally?

### CODE REVIEW

File: modules/users/client/components/ProfileViewBasics.js [hunk 1]
```diff
@@ -133,6 +133,20 @@ export default function ProfileViewBasics({ profile }) {
               )
             );
           })}
+        {/* Couchers.org */}
+        {profile.extSitesCouchers && (
+          <li className="social-profile">
+            <i className="social-profile-icon icon-fw icon-lg icon-bw"></i>
```
Comment (simison) [inline modules/users/client/components/ProfileViewBasics.js:139]: This icon will need updating:

Screenshot 2022-01-05 at 20 31 53

Comment (aapeliv) [reply modules/users/client/components/ProfileViewBasics.js:139]: Ah good point, thanks

**Comment by simison** [comment]
Overall testing well! Just icon will need updating.

image

Screenshot 2022-01-05 at 20 32 14

File: modules/users/client/components/ProfileViewBasics.js (hunk 1 above)
Comment (aapeliv) [reply modules/users/client/components/ProfileViewBasics.js]: Fixed

**Comment by aapeliv** [comment]
@simison: I think I fixed the review comment. But I'm not sure how to fix the linter error in CI?

**Comment by simison** [comment]
Linter fails for forks currently (https://github.com/Trustroots/trustroots/pull/2491 might fix it) but you can run it locally with:
```
npm ci
npm run lint
```

( I can also check before merging)

**Comment by aapeliv** [comment]
I can't run `npm ci` locally, that fails for me.

**Review by simison — APPROVED** [review]
Works well now, thank you!

image

Linter is happy locally, too.

**Comment by simison** [comment]
BTW side note that CI translation commit in this PR was from Github action I added a while back (https://github.com/Trustroots/trustroots/pull/2438) but looks like I need to adjust how it works with forks.

**Pull request merged**